docs/cli.md

```diff
@@ -296,6 +296,7 @@ viam data export --destination=<output path> --data-type=<output data type> [...
 viam data delete [...named args]
 viam data dataset add --dataset-id=<dataset-id> --file-ids=<file-id-or-ids> --location-id=<location-id> --org-id=<org-id> [...named args]
 viam data dataset remove --dataset-id=<dataset-id> --file-ids=<file-id-or-ids> --location-id=<location-id> --org-id=<org-id> [...named args]
+viam data dataset export --destination=<output-directory> --datasetFlagDatasetID=<dataset-id>
```

Comment:
destination already included export in "named arguments" @tahiyasalam LMK if this needs correcting, a bit confused about flags not being renamed to simple names like destination is

Reply:
Please make sure you install the latest version of the CLI and check it to get this information:

```
(viam-docs) ➜  viam-docs git:(viam-agent-full) ✗ viam dataset export
NAME:
   viam dataset export - download data from a dataset

USAGE:
   viam dataset export --destination=<destination> --dataset-id=<dataset-id> [other options] include-jsonl parallel

OPTIONS:
   --dataset-id value   dataset ID of the dataset to be downloaded
   --destination value  output directory for downloaded data
   --include-jsonl      option to include JSON Lines files for local testing (default: false)
   --parallel value     number of download requests to make in parallel (default: 100)

Error: Required flags "destination, dataset-id" not set
```

Alternatively, the information is also in [app.go](https://github.com/tahiyasalam/rdk/blob/main/cli/app.go#L686). The lines that set the name parameters are:
- https://github.com/tahiyasalam/rdk/blob/main/cli/dataset.go#L18
- https://github.com/tahiyasalam/rdk/blob/main/cli/app.go#L75
- etc.

You can search for the variable name and see where it is getting included to find out more.

Reply:
Thank you, I wasn't aware of this and was struggling to find. Will update!